### ISSUE TITLE
TYPO3 13

### ISSUE THREAD

**Issue by matin73** [mittwald/typo3-web2pdf#139]
... are there planes to have version for latest TYPO3 13?
Martin

**Comment by derhansen** [comment]
Yes, it is planned to provide a TYPO3 v13 compatible version of the extension in Q1/2025

**Comment by derhansen** [comment]
Version 5.0.0 including TYPO3 13.4 support has been published on packagist. TER will follow soon
